HorusPackets.py

```diff
@@ -1,55 +1,129 @@
 #!/usr/bin/env python2.7
 #
-#   Project Horus 
+#   Project Horus
 #   Packet Parsers
 #   Copyright 2015 Mark Jessop <vk5qi@rfhead.net>
 #
+#   Modifications by Will Anthony <willanth@gmail.com>
+#
 
-import time, struct, json, socket, httplib, crcmod, urllib, urllib2
+import time
+import struct
+import json
+import socket
+import httplib
+import crcmod
 from base64 import b64encode
 from hashlib import sha256
 from datetime import datetime
 
+#TODO import config file to get ports
 HORUS_UDP_PORT = 55672
 HORUS_OZIPLOTTER_PORT = 8942
 MAX_JSON_LEN = 2048
 TX_QUEUE_SIZE = 32
 
+
 # Packet Payload Types
 class HORUS_PACKET_TYPES:
+    """Types of Packets used for Horus"""
     PAYLOAD_TELEMETRY     = 0
     TEXT_MESSAGE          = 1
     CUTDOWN_COMMAND       = 2
     PARAMETER_CHANGE      = 3
     COMMAND_ACK           = 4
-    # Accept SSDV packets 'as-is'. https://ukhas.org.uk/guides:ssdv
-    SSDV_FEC              = 0x66
-    SSDV_NOFEC            = 0x67
 
-class HORUS_PAYLOAD_PARAMS:
+
+class HORUS_PAYLOAD_PARAMS:         #TODO include settings to enable/disable TDMA as well as set TDMA slot
+    """Horus Payload transmit/listen time"""
     PING                  = 0
     LISTEN_TIME           = 1
+    TDMA_ENABLE           = 2
+    TDMA_SLOT             = 3
+
 
 
 # Some utilities to use in other programs.
 def decode_payload_type(packet):
     # This expects the payload as an integer list. Convert it to one if it isn't already
+    """
+    Decodes the packet type.
+
+    Reads off the first value of the packet (type flag) and returns this.
+
+    Parameters
+    ----------
+    arg1 : list
+            RX'd radio packet
+
+    Returns
+    -------
+    int
+        Packet type value as defined in HORUS_PACKET_TYPES
```

Comment:
Guideline was taken from:
http://docs.python-guide.org/en/latest/writing/documentation/

"For more detailed documentation of code a popular style is the one used for the Numpy project, often called Numpy style docstrings"

For the purposes of future Sphinx integration.